clients/admin-ui/src/pages/login.tsx

```diff
@@ -71,7 +82,26 @@ const useLogin = () => {
   };
 
   if (token) {
-    router.push(INDEX_ROUTE);
+    const configWizardFlagIsActive = Flags.some(
+      (flag) => flag.name === "configWizardFlag" && flag.isActive
+    );
+
+    if (configWizardFlagIsActive) {
+      if (systems && systems.length > 0) {
```

Comment:
This will be false if the systems query is still loading, which I think means the page will render. The loading state should take this query into account.